streamer/src/quic.rs

```diff
@@ -117,13 +117,15 @@ pub enum QuicServerError {
 }
 
 pub struct EndpointKeyUpdater {
-    endpoint: Endpoint,
+    endpoints: Vec<Endpoint>,
 }
 
 impl NotifyKeyUpdate for EndpointKeyUpdater {
     fn update_key(&self, key: &Keypair) -> Result<(), Box<dyn std::error::Error>> {
         let (config, _) = configure_server(key)?;
-        self.endpoint.set_server_config(Some(config));
+        for endpoint in &self.endpoints {
```

Comment:
Btw i mechanically updated this and it appears to work. @ryleung-solana can you
double check that this is fine?

Reply:
Should be fine.